File: kotlin-ide/idea/testData/hierarchy/overrides/kotlinFunctionInTrait/main1.java
interface A extends T {
    @Override
    public void foo();
}

class B implements T {
    @Override
    public void foo() {

    }
}

class C implements A {
    @Override
    public void foo() {

    }
}

class D extends Z {
    @Override
    public void foo() {

    }
}

class S {

}

File: native/native.tests/testData/caches/ic/members/classModification/main/main.1.kt
import kotlin.test.*
import test.*

@Test
fun runTest() {
    val demo = Demo("a", "fixed")
    assertEquals("foo a", useFoo(demo))
    assertEquals("hidden changed a", useHidden(demo))
    assertEquals("demo field", useField(demo))
    assertEquals("fixed", useDefaultY(demo))
}
